src/components/UI/Skeletons/PlioListItemSkeleton.vue

```diff
@@ -0,0 +1,18 @@
+<template>
```

Comment:
guessing that needs to be changed.

Reply:
Yes will edit this, looks very weird right now